src/current/_includes/v23.2/sql/select-lock-strengths.md

```diff
@@ -0,0 +1,5 @@
+- `SELECT FOR UPDATE` obtains an *exclusive lock* on each qualifying row, blocking concurrent writes and locking reads on the row. Only one transaction can hold an exclusive lock on a row at a time, and only the transaction holding the exclusive lock can write to the row. {% if page.name == "read-committed.md" %}For an example, see [Reserve rows for updates using exclusive locks](#reserve-rows-for-updates-using-exclusive-locks).{% endif %}
+
+- `SELECT FOR SHARE` obtains a *shared lock* on each qualifying row, blocking concurrent writes and **exclusive** locking reads on the row. Multiple transactions can hold a shared lock on a row at the same time. When multiple transactions hold a shared lock on a row, none can write to the row. A shared lock grants transactions mutual read-only access to a row, and ensures that they read the latest value of the row. {% if page.name == "read-committed.md" %}For an example, see [Reserve values using shared locks](#reserve-row-values-using-shared-locks).{% endif %}
+
+When a `SELECT FOR UPDATE` or `SELECT FOR SHARE` read is issued on a row, only the latest value of the row is returned to the client. Under {% if page.name == "read-committed.md" %}`READ COMMITTED`{% else %}[`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}){% endif %} isolation, neither statement will block concurrent, non-locking reads.
```

Comment:
This is a great paragraph. nit: Might consider using "version" instead of "value" in the first sentence. (And also in the FOR SHARE paragraph.)